monkey/monkey_island/cc/services/configuration/utils.py

```diff
@@ -1,6 +1,29 @@
-from common.config_value_paths import INACCESSIBLE_SUBNETS_PATH
+from common.config_value_paths import (
+    INACCESSIBLE_SUBNETS_PATH,
+    LM_HASH_LIST_PATH,
+    NTLM_HASH_LIST_PATH,
+    PASSWORD_LIST_PATH,
+    SSH_KEYS_PATH,
+    USER_LIST_PATH,
+)
 from monkey_island.cc.services.config import ConfigService
 
 
 def get_config_network_segments_as_subnet_groups():
     return [ConfigService.get_config_value(INACCESSIBLE_SUBNETS_PATH)]
+
+
+def get_config_propagation_credentials():
+    return {
+        "exploit_user_list": ConfigService.get_config_value(USER_LIST_PATH, should_decrypt=False),
+        "exploit_password_list": ConfigService.get_config_value(
+            PASSWORD_LIST_PATH, should_decrypt=False
+        ),
+        "exploit_lm_hash_list": ConfigService.get_config_value(
+            LM_HASH_LIST_PATH, should_decrypt=False
+        ),
+        "exploit_ntlm_hash_list": ConfigService.get_config_value(
+            NTLM_HASH_LIST_PATH, should_decrypt=False
+        ),
+        "exploit_ssh_keys": ConfigService.get_config_value(SSH_KEYS_PATH, should_decrypt=False),
+    }
```

Comment:
Move this to the `ConfigurationService`. Resources depend on services; we should minimize other dependencies.

The services are part of the core application. These resources are just a way of accessing those services. These lines are not clearly drawn at the present time, but I'm hoping to fix them in the next 6 months.